packages/blade/src/components/Radio/RadioIcon.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import React from 'react';
+import { useTheme } from '~components/BladeProvider';
+import Svg from '~components/Icons/_Svg';
```

Comment:
nit: We should do named exports here too since we've decided to do it everywhere 😅 

We can do this later as well. Not in this PR's scope

Reply:
Yeah let's do this on a separate PR.